[2022-09-17 05:50:53,520] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: yellow_taxi_data_v2.local_to_gcs_task scheduled__2019-06-02T06:00:00+00:00 [queued]>
[2022-09-17 05:50:53,533] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: yellow_taxi_data_v2.local_to_gcs_task scheduled__2019-06-02T06:00:00+00:00 [queued]>
[2022-09-17 05:50:53,533] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-09-17 05:50:53,534] {taskinstance.py:1239} INFO - Starting attempt 1 of 2
[2022-09-17 05:50:53,534] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-09-17 05:50:53,548] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): local_to_gcs_task> on 2019-06-02 06:00:00+00:00
[2022-09-17 05:50:53,555] {standard_task_runner.py:52} INFO - Started process 4394 to run task
[2022-09-17 05:50:53,562] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'yellow_taxi_data_v2', 'local_to_gcs_task', 'scheduled__2019-06-02T06:00:00+00:00', '--job-id', '19', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion_gcs_dag.py', '--cfg-path', '/tmp/tmpxocw3bx_', '--error-file', '/tmp/tmpczfch26g']
[2022-09-17 05:50:53,563] {standard_task_runner.py:77} INFO - Job 19: Subtask local_to_gcs_task
[2022-09-17 05:50:53,653] {logging_mixin.py:109} INFO - Running <TaskInstance: yellow_taxi_data_v2.local_to_gcs_task scheduled__2019-06-02T06:00:00+00:00 [running]> on host 10d063af5ba7
[2022-09-17 05:50:53,706] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2022-09-17 05:50:53,734] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=yellow_taxi_data_v2
AIRFLOW_CTX_TASK_ID=local_to_gcs_task
AIRFLOW_CTX_EXECUTION_DATE=2019-06-02T06:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2019-06-02T06:00:00+00:00
[2022-09-17 05:50:56,764] {python.py:175} INFO - Done. Returned value was: None
[2022-09-17 05:50:56,779] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=yellow_taxi_data_v2, task_id=local_to_gcs_task, execution_date=20190602T060000, start_date=20220917T055053, end_date=20220917T055056
[2022-09-17 05:50:56,831] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-09-17 05:50:56,867] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2022-09-20 15:32:37,432] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: yellow_taxi_data_v2.local_to_gcs_task scheduled__2019-06-02T06:00:00+00:00 [queued]>
[2022-09-20 15:32:37,450] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: yellow_taxi_data_v2.local_to_gcs_task scheduled__2019-06-02T06:00:00+00:00 [queued]>
[2022-09-20 15:32:37,450] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-09-20 15:32:37,451] {taskinstance.py:1239} INFO - Starting attempt 1 of 2
[2022-09-20 15:32:37,451] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-09-20 15:32:37,470] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): local_to_gcs_task> on 2019-06-02 06:00:00+00:00
[2022-09-20 15:32:37,479] {standard_task_runner.py:52} INFO - Started process 510 to run task
[2022-09-20 15:32:37,490] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'yellow_taxi_data_v2', 'local_to_gcs_task', 'scheduled__2019-06-02T06:00:00+00:00', '--job-id', '80', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion_gcs_dag.py', '--cfg-path', '/tmp/tmpe1scz134', '--error-file', '/tmp/tmpgwq4slde']
[2022-09-20 15:32:37,492] {standard_task_runner.py:77} INFO - Job 80: Subtask local_to_gcs_task
[2022-09-20 15:32:37,588] {logging_mixin.py:109} INFO - Running <TaskInstance: yellow_taxi_data_v2.local_to_gcs_task scheduled__2019-06-02T06:00:00+00:00 [running]> on host 788dccad41b9
[2022-09-20 15:32:37,666] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2022-09-20 15:32:37,706] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=yellow_taxi_data_v2
AIRFLOW_CTX_TASK_ID=local_to_gcs_task
AIRFLOW_CTX_EXECUTION_DATE=2019-06-02T06:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2019-06-02T06:00:00+00:00
[2022-09-20 15:32:38,661] {python.py:175} INFO - Done. Returned value was: None
[2022-09-20 15:32:38,697] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=yellow_taxi_data_v2, task_id=local_to_gcs_task, execution_date=20190602T060000, start_date=20220920T153237, end_date=20220920T153238
[2022-09-20 15:32:38,749] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-09-20 15:32:38,955] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-09-21 01:33:45,579] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: yellow_taxi_data_v2.local_to_gcs_task scheduled__2019-06-02T06:00:00+00:00 [queued]>
[2022-09-21 01:33:45,595] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: yellow_taxi_data_v2.local_to_gcs_task scheduled__2019-06-02T06:00:00+00:00 [queued]>
[2022-09-21 01:33:45,595] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-09-21 01:33:45,595] {taskinstance.py:1239} INFO - Starting attempt 1 of 2
[2022-09-21 01:33:45,595] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-09-21 01:33:45,615] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): local_to_gcs_task> on 2019-06-02 06:00:00+00:00
[2022-09-21 01:33:45,623] {standard_task_runner.py:52} INFO - Started process 27655 to run task
[2022-09-21 01:33:45,634] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'yellow_taxi_data_v2', 'local_to_gcs_task', 'scheduled__2019-06-02T06:00:00+00:00', '--job-id', '252', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion_gcs_dag.py', '--cfg-path', '/tmp/tmp250xeg5p', '--error-file', '/tmp/tmpmvw3nayi']
[2022-09-21 01:33:45,635] {standard_task_runner.py:77} INFO - Job 252: Subtask local_to_gcs_task
[2022-09-21 01:33:45,738] {logging_mixin.py:109} INFO - Running <TaskInstance: yellow_taxi_data_v2.local_to_gcs_task scheduled__2019-06-02T06:00:00+00:00 [running]> on host 788dccad41b9
[2022-09-21 01:33:45,799] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2022-09-21 01:33:45,825] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=yellow_taxi_data_v2
AIRFLOW_CTX_TASK_ID=local_to_gcs_task
AIRFLOW_CTX_EXECUTION_DATE=2019-06-02T06:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2019-06-02T06:00:00+00:00
[2022-09-21 01:33:46,755] {python.py:175} INFO - Done. Returned value was: None
[2022-09-21 01:33:46,772] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=yellow_taxi_data_v2, task_id=local_to_gcs_task, execution_date=20190602T060000, start_date=20220921T013345, end_date=20220921T013346
[2022-09-21 01:33:46,813] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-09-21 01:33:46,852] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
